starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -52,4 +52,14 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+
+  // Test 'Create' DAO method
+  @Test
+  public void create() {
+
+    Inventory inventory = new Inventory();
+    Inventory createdInventory = this.inventoryDAO.create(inventory);
+    String id = createdInventory.getId();
+    Assert.assertNotNull(id);
```

Comment:
For most null assertions, we would like to see if our DAO method returns an object or a null value.

- `Assert.assertNotNull` should check `createdInventory` to see if we saved an inventory
- Line 62 (`String id = createdInventory.getId();`) should be removed
- After the `Assert.assertNotNull`, add assertions comparing values between `inventory` and `createdInventory` to see if they are equal (name/productType) or not equal (id)